src/server/game/Maps/Map.cpp

```diff
@@ -1091,6 +1092,38 @@ void Map::DynamicObjectRelocation(DynamicObject* dynObj, float x, float y, float
     ASSERT(integrity_check == old_cell);
 }
 
+void Map::AreaTriggerRelocation(AreaTrigger* at, float x, float y, float z, float orientation)
+{
+    Cell integrity_check(at->GetPositionX(), at->GetPositionY());
+    Cell old_cell = at->GetCurrentCell();
```

Comment:
old_cell contains 0 and will trigger assertion
Map::AddToGrid block with areatrigger specialisation is missing?